lib/rijksbot.rb

```diff
@@ -29,7 +29,7 @@ def random_image
   end
 
   def get_image(search_term)
-    api_key = ENV["rijksmuseum_api_key"]
+    api_key = ENV["RIJKSMUSEUM_API_KEY"]
```

Comment:
💯

Reply:
oh and if you can invalidate the old api key, since it's still in the repository history